ts/Stock/Indicators/DMI/DMIOptions.d.ts

```diff
@@ -0,0 +1,46 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type CSSObject from '../../../Core/Renderer/CSSObject';
+import type {
+    SMAOptions,
+    SMAParamsOptions
+} from '../SMA/SMAOptions';
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+export interface DMIOptions extends SMAOptions {
+    params?: DMIParamsOptions;
+    tooltip?: Highcharts.TooltipOptions;
+    plusDILine?: DMILineOptions;
+    minusDILine?: DMILineOptions;
+}
+
+export interface DMILineOptions {
+    styles?: CSSObject;
+}
+
+export interface DMIZonesOptions {
+    startIndex?: number;
+}
+
```

Comment:
As far as I can see, this interface is not in use and can be removed.

Reply:
Thanks!